source/webapp/sites/all/modules/custom/checkbook_api/config/spending.json

```diff
@@ -86,7 +86,7 @@
                         "Check Amount":"check_amount",
                         "Expense Category":"expenditure_object_name",
                         "Contract ID":"reference_document_number",
-                        "Contract Purpose":"purpose",

```

Comment:
this doesn't need to be reverted I suppose

Reply:
yes , this is not needed.